integration_test/testdata/prometheus/setup_json_exporter.sh

```diff
@@ -1,8 +1,17 @@
 set -e
 
 # Download JSON exporter and extract to /opt/
+unset ARCH
+case "$(uname -m)" in
+  "x86_64")
+    ARCH="amd64"
+    ;;
+  "aarch64")
+    ARCH="arm64"
+    ;;
+esac
```

Comment:
`dpkg` isn't portable though, so that simplicity would be offset by the complexity of finding equivalents on other distros...

The map version is nice because it's still portable yet simpler than what I had already, so I'll go with that for now unless there's an even simpler alternative that works on all distros.

Reply:
You're right, I'm so used to Debian using its own architecture constants that I didn't notice that this script is cross-distro.